src/Core/Entity/ContactRepository.php

```diff
@@ -13,10 +13,13 @@
 
 use Ushahidi\Core\Entity\Repository\EntityGet;
 use Ushahidi\Core\Entity\Repository\EntityExists;
+use Ushahidi\Core\Usecase\CreateRepository;
+use Ushahidi\Core\Usecase\UpdateRepository;
 
 interface ContactRepository extends
 	EntityGet,
-	EntityExists
+	EntityExists,
+	CreateRepository
```

Comment:
I've added this to a few repos. It much easier to mock for tests is everything goes into a single interface. Hopefully it'll make DI better in future too